src/cftlib/resources/scripts/start.sh

```diff
@@ -15,4 +15,25 @@ loadSecret "LAUNCH_DARKLY_SDK_KEY" "wa-aat" "ld-secret"
 
 echo "Secret fetching complete"
 
+az acr login --name hmctspublic --subscription 8999dec3-0104-4a27-94ee-6588559729d1
+
+echo "Switch to definitions folder"
+cd definitions
+git submodule update --init
+git checkout master
+git pull
+echo "Switch to parent folder"
+cd ../
+
+HOST_ENTRY="127.0.0.1    rse-idam-simulator"
+
+if grep -q "$HOST_ENTRY" /etc/hosts; then
+  echo "Entry already exists in /etc/hosts"
+else
+  sudo bash -c "echo '$HOST_ENTRY' >> /etc/hosts"
+fi
```

Comment:
Did we miss the dnsflush steps in previous readme?

Reply:
Ah good catch, added